app/models/application_reference.rb

```diff
@@ -58,4 +58,8 @@ def replace_referee_at
 
     TimeLimitCalculator.new(rule: :replace_referee_by, effective_date: requested_at).call.second
   end
+
+  def response_overdue?
```

Comment:
We can use https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/models/application_reference.rb#L62-L68 now.